src/Workspaces/Remote/ServiceHub/Host/AssetProvider.cs

```diff
@@ -129,58 +135,87 @@ public async ValueTask SynchronizeProjectAssetsAsync(ProjectStateChecksums proje
         // GetAssetAsync call will most likely cache hit. it is most likely since we might change cache heuristic in
         // future which make data to live a lot shorter in the cache, and the data might get expired before one actually
         // consume the data. 
-        using (Logger.LogBlock(FunctionId.AssetService_SynchronizeProjectAssetsAsync, Checksum.GetProjectChecksumsLogInfo, projectChecksums, cancellationToken))
+        using (Logger.LogBlock(FunctionId.AssetService_SynchronizeProjectAssetsAsync, message: null, cancellationToken))
```

Comment:
>message: null [](http://example.com/codeflow?start=86&length=13)

Intentionally got rid of this I'm guessing. If so, is GetProjectChecksumsLogInfo still used?

Reply:
probably not.  i'll remove.